packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md

```diff
@@ -54,63 +54,180 @@ Therefore, when dealing with Custom Roles, these three entities always exist:
 
 ### Resource sets
 
-A Resource Set is simply a collection of resources. The following resources are currently supported:
+A Resource Set is simply a collection of resources. There are two types of resource identifiers, resources can either be identified by Okta Resource Names (ORN) or by a REST URL format.
 
-* All Users
-* All Groups
-* A specific Group
-* All Users within a specific Group
-* All Apps
-* All Apps of the same type
-* A specific App
+#### Okta Resource Name (ORN)
 
-### Identifiers
+<ApiLifecycle access="ea" />
 
-#### Resource identifiers
+Okta Resource Names (ORN) is the primary resource identifier. ORNs uniquely identify Okta resources. Currently ORNs are available in early access.
 
-To specify a resource targeted by a Resource Set, you can use the REST URL of the corresponding Okta API:
+##### ORN Format
 
-* [All Users](/docs/reference/api/users/#list-users)
+ORNs generally follow a format, however each type of resource have their own specific format.
 
-  ``` http
-  https://${yourOktaDomain}/api/v1/users
-  ```
-
-* [All Groups](/docs/reference/api/groups/#list-groups)
-
-  ``` http
-  https://${yourOktaDomain}/api/v1/groups
-  ```
-
-* [All Apps](/docs/reference/api/apps/#list-applications)
+`orn:${partition}:${service}:${tenantId}:${objectType}:${objectId}`
 
-  ``` http
-  https://${yourOktaDomain}/api/v1/apps
-  ```
+###### partition
 
-* [A specific Group](/docs/reference/api/groups/#get-group)
+The partition is specific to your Okta environment. The following are the supported partitions.
 
-  ``` http
-  https://${yourOktaDomain}/api/v1/groups/${targetGroupId}
-  ```
+| Partition               |  ORN partition value  |
+| ----------------------- | --------------------- |
+| Preview environment     | oktapreview           |
+| Every other environment | okta                  | 
 
-* [All Users within a specific Group](/docs/reference/api/groups/#list-group-members)
+###### service
 
-  ``` http
-  https://${yourOktaDomain}/api/v1/groups/${targetGroupId}/users
-  ```
-
-* [All Apps of specific type](/docs/reference/api/apps/#list-apps-by-name)
-
-  ``` http
-  https://${yourOktaDomain}/api/v1/apps/?filter=name+eq+\"${targetAppType}\"
-  ```
-
-* [A specific App](/docs/reference/api/apps/#get-application)
-
-  ``` http
-  https://${yourOktaDomain}/api/v1/apps/${targetAppId}
-  ```
+  The service in which the resource belongs to. Each resource only belongs to one service.
+  The following are the supported services.
+
+| Service           |  ORN service value    |
+| ----------------- | --------------------- |
+| Directory         | directory             | 
+| Identity Provider | idp                   |
```

Comment:
We should list all services we support today, tagging those behind EA with EA tag. We should also mention resource type (objectType) supported by each service in a separate column